src/pages/resultsView/pathwayMapper/PathWayMapperContainer.tsx

```diff
@@ -27,8 +79,20 @@ const PathWayMapperContainer: React.FunctionComponent<IPathwayMapperContainerPro
             activeTab:
                 urlWrapper.query.pathways_source ||
                 ResultsViewPathwaysSubTab.PATHWAY_MAPPER,
+            tooltipContent: null as DatabaseItem[] | null,
+            isFetchingTooltip: false,
+            tooltipData: makeRemoteData(),
         }));
 
+        autorun(() => {
```

Comment:
you don't need this stuff here, you can just reference tooltipData.result, toooltipData.isComplete, tooltipData.isPending, tooltipData.isError in your UI.  Because these properties are mobx observables and the component is wrapped with observer(), the render will automatically be fired.